lib/packwerk/commands/detect_stale_violations_command.rb

```diff
@@ -10,54 +10,56 @@
 require "packwerk/commands/offense_progress_marker"
 
 module Packwerk
-  class DetectStaleViolationsCommand
-    extend T::Sig
-    include OffenseProgressMarker
-    Result = Struct.new(:message, :status)
-
-    def initialize(files:, configuration:, run_context: nil, progress_formatter: nil, reference_lister: nil)
-      @configuration = configuration
-      @run_context = run_context
-      @reference_lister = reference_lister
-      @progress_formatter = progress_formatter
-      @files = files
-    end
+  module Commands
```

Comment:
I like that you moved the class into its own namespace, which makes it cleaner :)